events-application-rest-api/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -65,4 +65,8 @@ databaseChangeLog:
               - column:
                   name: date
                   type: timestamp
-
+              - column:
+                  name: privacyoption
+                  type: boolean
+                  constraints:
+                    nullable: false
```

Comment:
Can you add a `default` constraint to it? By default, it's better if the event is _not_ private.